Observer/Order/Save.php

```diff
@@ -134,7 +139,8 @@ public function execute(Observer $observer)
                 if ($nostoCustomer instanceof NostoCustomer === false) {
                     return;
                 }
-                $orderService = new OrderConfirm($nostoAccount);
+                $store = $order->getStore();
+                $orderService = new OrderConfirm($nostoAccount, $this->nostoHelperUrl->getActiveDomain($store));
```

Comment:
The account seems to come from the scope while the domain is deduced from the order?

Reply:
Good point! I believe both should be from `Order`.  Olsi can you refactor this one: https://github.com/Nosto/nosto-magento2/blob/391354b13f30e84b7424f3d97e3414b2c0919b9a/Observer/Order/Save.php#L132